Foundation/GTMSQLite.m

```diff
@@ -531,7 +531,8 @@ static void UpperLower8(sqlite3_context *context, int argc, sqlite3_value **argv
   }
 
   _GTMDevAssert(userArgs->textRep == SQLITE_UTF8,
-                @"Received non UTF8 encoding in UpperLower8");
+                @"Received non UTF8 encoding in UpperLower8, %d",
```

Comment:
nit, you divide your format string with a "," here and a colon above. Standardize?

Reply:
done